app/src/main/java/com/T05/krowdtrialz/ui/ExperimentDetailsOwnerActivity.java

```diff
@@ -0,0 +1,16 @@
+package com.T05.krowdtrialz.ui;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+
+import com.T05.krowdtrialz.R;
+
+public class ExperimentDetailsOwnerActivity extends AppCompatActivity {
+
+    @Override
```

Comment:
I also wonder if we might want some packages for these too. Not sure what it might be called though.

Reply:
Maybe on Sunday we can decide together on a hierarchy to organize files. We can move stuff around then after we decide.